### PR TITLE
Added documentation on halting the action chain

### PR DESCRIPTION
**Pull request by cannikin**
In regards to @sstephenson's answer here: https://discourse.stimulusjs.org/t/possible-to-stop-calls-with-multiple-actions/628 I thought I'd add this to the Reference manual for future generations!

### CODE REVIEW

**Comment by brendon** [comment]
This looks like a useful documentation upgrade :)

**Comment by cannikin** [comment]
Thanks! The PR is almost two years old now but no one wants to merge it. :(

**Comment by brendon** [comment]
Hopefully this will act as a reminder :)

File: docs/reference/actions.md [hunk 1]
```diff
@@ -121,6 +121,15 @@ It's common for any given element to have many actions. For example, the followi
 
 When an element has more than one action for the same event, Stimulus invokes the actions from left to right in the order that their descriptors appear.
 
+The action chain can be stopped at any point by calling `Event#stopImmediatePropagation()` within an action. Any addtional actions to the right will be ignored:
```
Comment (JuanVqz) [inline docs/reference/actions.md:124]: I noticed a typo here Additional instead of addtional

**Comment by cannikin** [comment]
Thanks @dhh!

**Pull request merged**